src/popup/components/TransactionLabel.vue

```diff
@@ -35,7 +35,6 @@
           v-if="externalLabel.length"
         >
```

Comment:
Currently in case the type failed to be found (another discussion, should be fixed on sdk update), the` txType.value` will return `false`, so next line should be adjust
https://github.com/aeternity/superhero-wallet/blob/fd2a6d7067aabb6cae32f45a07c53868ad5e85d1/src/popup/components/TransactionLabel.vue#L234
to
`if(txType.value && txType.value.includes('name'))`